modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -289,7 +289,10 @@
 (defmethod ->rvalue ::not [[_ value]]
   {$not (->rvalue value)})
 
-(defmulti ^:private compile-filter mbql.u/dispatch-by-clause-name-or-class)
+(defmulti compile-filter
+  "Compile an mbql filter clause to datastructures suitable to query mongo. Note this is not the whole query but just
+  compiling the \"where\" clause equivalent."
```

Comment:
Can you add `:arglists` metadata to the `defmulti`? It's nice to be able to know what arguments the multimethod takes without having to go hunt down a `defmethod` form for it. Editors can show this info too